### ISSUE TITLE
Preview Pane not very touch friendly

### ISSUE THREAD

**Issue by davidc86** [sillsdev/ptx2pdf#1018]
The PDF Preview Pane is great, but it is not very touch friendly. In fact, it usually ignores touches.
On occasion I got the + - buttons to register my touch, but it generally only works if I touch on another window first, and then touch on the Title bar of the Preview Pane and then touch the + or - button. (Maybe because focus is still held by the window frame?)

Again, not a big bug, but something to be aware of, and if it is easy to make the pane touch-aware, that'd be cool. (Helpful when working on the road and messing with the mouse or a touch pad is cumbersome.)

**Comment by markpenny** [comment]
To be honest, I've never thought about the touch interface for PTXprint. (Now my screen has fingerprints all over it ;-) !!! but I can see what works and what doesn't.
I have contemplated doing pages with navigation buttons a bit like this:
|**<**   **<**   Page **73** / 146   **>**  **>**|      where 73 is editable and then the **>>** as well for show next underfilled page [yet to get that working for 1-col publications]
MH did warn me that as soon as we came up with a PDF Viewer, people would expect it to do everything a normal PDF viewer can do, and more. He was right. Someone else already said they were missing the Find Text option!
BTW - how do you do a right-click in the touch interface?

**Comment by davidc86** [comment]
You ask: - how do you do a right-click in the touch interface?
Use a LONG Press. (Just touch something longer than usual and Windows will pop open the rt-click menu.)

MH did warn me that as soon as we came up with a PDF Viewer, people would expect it to do everything a normal PDF viewer can do, and more. He was right. Someone else already said they were missing the Find Text option!

Yes, sorry. I've fired off a pile of those whiney kinds of "enhancement" requests myself. 

But just so you know, even if nothing else is ever done to the PDF Preview Pane, it currently is a HUGE improvement over the old way. And your ideas for tweaking paragraphs etc in-place is a wonderful, powerful, enhancement to PTXprint's functionality. So don't be overwhelmed by our whining. It is exciting stuff.

**Comment by davidc86** [comment]
Well, v 2.7.13 seems to have disabled Touch on all windows including the main PTXprint window. Was this intentional?

**Comment by markpenny** [comment]
No, it is working fine for me; so please check your setup/environment and try again.

**Comment by davidc86** [comment]
Weird!
I just checked in again and nope, Touch doesn't work on any button or drop-down or text box in either the PTXprint main menu or the PDF Viewer. 

All other apps respond to Touch on my laptop.  

Closing and opening PTXprint doesn't help.

I have no idea what might be going on now. 

WAIT! 
I used my digital pen, and the first click did nothing, but finally I got a drop-down to open. Then Touch started working.
Same thing with PDF Viewer. At first it wouldn't respond, and finally the pen got it to respond, and now Touch works. 

REALLY WEIRD. I've never seen an app not respond to Touch like that.

No clue. I suppose we can close this until further notice.
